pkg/query/v2/logical/plan_tablescan.go

```diff
@@ -0,0 +1,195 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
```

Comment:
This plan looks like a subset of indexscan. Could we merge them? or use another structure

Reply:
Removed. It has been merged into `IndexScan` now.